.coveragerc

```diff
@@ -317,6 +317,7 @@ omit =
     homeassistant/components/glances/sensor.py
     homeassistant/components/gntp/notify.py
     homeassistant/components/goalfeed/*
+    homeassistant/components/goalzero/*
```

Comment:
You can't exclude all files. We require, at minimum, tests for `config_flow.py` with a 100% test coverage.

Reply:
@frenck I now have tests which pass locally but the online tests show less than 100% coverage. Group 2 fails outright. I'm at a loss on how to fix this

Reply:
100% now but python 3.7 group 2 fails. Compatibility issue? 😒